theme/templates/newbase.html

```diff
@@ -14,20 +14,28 @@
         {% block meta_title %}{% endblock %}{% if settings.SITE_TITLE %} | {{ settings.SITE_TITLE }}{% endif %}</title>
     <link rel="shortcut icon" href="/static/img/favicon.png">
 
```

Comment:
newbase will eventually replace base as pages are carefully migrated to it one-by-one over a very long period of time